pkg/features/kube_features.go

```diff
@@ -916,7 +917,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	StatefulSetAutoDeletePVC:                       {Default: false, PreRelease: featuregate.Alpha},
 	TopologyAwareHints:                             {Default: false, PreRelease: featuregate.Beta},
 	PodAffinityNamespaceSelector:                   {Default: true, PreRelease: featuregate.Beta},
-	ServiceLoadBalancerClass:                       {Default: true, PreRelease: featuregate.Beta},
+	ServiceLoadBalancerClass:                       {Default: true, PreRelease: featuregate.GA},
```

Comment:
I think that the convention is 

when moving features to GA, we should:

> set LockToDefault: true and add a comment tracking removal in 2 releases (1.22 in this case)
> remove checks for .Enabled(features.RunAsGroup) (since it is locked to true)
> ensure API docs no longer indicate it is a beta feature (I think this is already done for this field)
> ensure e2e test coverage exists

https://github.com/kubernetes/kubernetes/pull/94641#discussion_r485622926